platform/android/java/lib/src/org/godotengine/godot/input/GodotGestureHandler.kt

```diff
@@ -197,8 +197,13 @@ internal class GodotGestureHandler : SimpleOnGestureListener(), OnScaleGestureLi
 		if (event.actionMasked == MotionEvent.ACTION_UP) {
 			nextDownIsDoubleTap = false
 			GodotInputHandler.handleMotionEvent(event)
```

Comment:
Also, is this correct, that a ACTION_UP should trigger a motion event?

Reply:
That's correct, this correspond to the release of the double tap event.